src/System.Private.ServiceModel/tests/Scenarios/ServiceModel.Scenario.Tests.builds

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="Binding\Custom\Binding.Custom.Tests.csproj" />
+    <Project Include="Binding\Http\Binding.Http.Tests.csproj" />
```

Comment:
Can we change all these into a single <Project Include=**\*.csproj />?  I'd prefer not to have to maintain a list of tests whenever we add a new test project.